zetaclient/chains/bitcoin/observer/inbound_test.go

```diff
@@ -152,41 +161,6 @@ func TestAvgFeeRateBlock828440Errors(t *testing.T) {
 	})
 }
 
-func TestCalcDepositorFee828440(t *testing.T) {
-	// load archived block 828440
-	var blockVb btcjson.GetBlockVerboseTxResult
-	testutils.LoadObjectFromJSONFile(
-		t,
-		&blockVb,
-		path.Join(TestDataDir, testutils.TestDataPathBTC, "block_trimmed_8332_828440.json"),
-	)
-	avgGasRate := float64(32.0)
-	// #nosec G115 test - always in range
-
-	gasRate := int64(avgGasRate * clientcommon.BTCOutboundGasPriceMultiplier)
-	dynamicFee828440 := bitcoin.DepositorFee(gasRate)
-
-	// should return default fee if it's a regtest block
-	fee := bitcoin.CalcDepositorFee(&blockVb, 18444, &chaincfg.RegressionNetParams, log.Logger)
-	require.Equal(t, bitcoin.DefaultDepositorFee, fee)
-
-	// should return dynamic fee if it's a testnet block
-	fee = bitcoin.CalcDepositorFee(&blockVb, 18332, &chaincfg.TestNet3Params, log.Logger)
-	require.NotEqual(t, bitcoin.DefaultDepositorFee, fee)
-	require.Equal(t, dynamicFee828440, fee)
-
-	// mainnet should return default fee before upgrade height
-	blockVb.Height = bitcoin.DynamicDepositorFeeHeight - 1
-	fee = bitcoin.CalcDepositorFee(&blockVb, 8332, &chaincfg.MainNetParams, log.Logger)
-	require.Equal(t, bitcoin.DefaultDepositorFee, fee)
-
-	// mainnet should return dynamic fee after upgrade height
-	blockVb.Height = bitcoin.DynamicDepositorFeeHeight
-	fee = bitcoin.CalcDepositorFee(&blockVb, 8332, &chaincfg.MainNetParams, log.Logger)
-	require.NotEqual(t, bitcoin.DefaultDepositorFee, fee)
-	require.Equal(t, dynamicFee828440, fee)
-}
-
 func TestGetSenderAddressByVin(t *testing.T) {
```

Comment:
Curious why is this test in inbounds_test and not observer_test , considering the function is defined in observer.go

Reply:
makes sense. I moved it to `inbound.go` as it's intended for inbound observation https://github.com/zeta-chain/node/pull/2899/commits/0e2a316e44c9400963d540ebbd11f93f11f0ae5f